static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -708,26 +766,20 @@ require.def(
                 throw "ApiError: " + errorMessage;
             },
 
-            _setDisplayFullScreenForVideo: function() {
```

Comment:
Would be interested to know why this is no longer necessary, but I assume testing proved it isn't.